cmd/kad/main.go

```diff
@@ -0,0 +1,91 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	"go.uber.org/zap"
+
+	"storj.io/storj/pkg/node"
+	"storj.io/storj/pkg/overlay"
+	"storj.io/storj/pkg/provider"
+)
+
+var (
+	rootCmd = &cobra.Command{
+		Use:   "inspector",
+		Short: "CLI for interacting with Storj Kademlia network",
+	}
+	getNodeCmd = &cobra.Command{
+		Use:   "get",
+		Short: "get node with `id`",
+		RunE:  GetNode,
+	}
+)
+
+// Inspector gives access to kademlia and overlay cache
+type Inspector struct {
+	overlay  overlay.Client
+	identity *provider.FullIdentity
+}
+
+func identity() (*provider.FullIdentity, error) {
+	ca, err := provider.NewTestCA(context.Background())
+	if err != nil {
+		zap.S().Errorf("Failed to create certificate authority: ", zap.Error(err))
+		os.Exit(1)
+	}
+	identity, err := ca.NewIdentity()
+	if err != nil {
+		zap.S().Errorf("Failed to create full identity: ", zap.Error(err))
+		os.Exit(1)
+	}
+	return identity, nil
+}
+
+// overlay returns an overlay client
+func newOverlayClient(identity *provider.FullIdentity, address string) (overlay.Client, error) {
```

Comment:
What the benefit of wrapping a single call here?

Reply:
Originally it did some more setup for the client, this was an artifact of some refactoring, so I'll pull it out.